packaging/debian-sid/changelog

```diff
@@ -1,3 +1,35 @@
+snapd (2.45.1-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1875071
+    - data/selinux: allow checking /var/cache/app-info
+    - cmd/snap-confine: add support for libc6-lse
+    - interfaces: miscellanious policy updates xlv
```

Comment:
this is a typo apparently and is breaking the spellchecking done by tests, s/i/e/